lang/en/texts/nutriscore.html

```diff
@@ -3,38 +3,27 @@ <h1>Compare the nutrition quality of food products with the Nutri-Score!</h1>
 <h2>What is Nutri-Score?</h2>
 
 <div class="row">
-
-  <div class="small-6 medium-6 large-8 xlarge-10 columns">
-
-   <img src="https://static.openfoodfacts.org/images/misc/nutriscore-a.png" alt="NutriScore">
-
-  <p>The Nutri-Score is a logo that shows the nutritional quality of food products with A to E grades. With the NutriScore, products can be easily and quickly compared.</p>
-    
+<div class="small-6 medium-6 large-8 xlarge-10 columns">
+<img src="https://static.openfoodfacts.org/images/misc/nutriscore-a.png" alt="NutriScore">
+<p>The Nutri-Score is a logo that shows the nutritional quality of food products with A to E grades. With the NutriScore, products can be easily and quickly compared.</p>
 Open Food Facts has started computing Nutri-Score in 2014 on its pioneer mobile app and website, while it was only a research paper, and has accompanied its growth from the "5 color score" to the "Nutri-Score" you see in supermarkets today.
-This systemic impact was noticed by Santé publique France which is now supporting Open Food Facts in its mission to help everyone make more informed food choices.
-    For more on the history of Open Food Facts and the Eco-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>.
-
-  </div>
-
+  However, the Nutri-Score is not affixed to all products and Open Food Facts allows you to have this information by scanning the product. Public Health France has supported OpenFoodFacts since 2018 to facilitate consumer information. For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>.
```

Comment:
```suggestion
  However, the Nutri-Score is not affixed to all products and Open Food Facts allows you to have this information by scanning the product. Public Health France has supported Open Food Facts since 2018 to facilitate consumer information. For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>.
```